packages/discord.js/src/client/actions/MessageCreate.js

```diff
@@ -6,7 +6,11 @@ const Events = require('../../util/Events');
 class MessageCreateAction extends Action {
   handle(data) {
     const client = this.client;
-    const channel = this.getChannel({ id: data.channel_id, guild_id: data.guild_id, author: data.author });
+    const channel = this.getChannel({
+      id: data.channel_id,
+      author: data.author,
+      ...(data.guildId && { guild_id: data.guild_id }),
```

Comment:
this will not include it if guildId is null
also, data.guild_id not data.guildId